### ISSUE TITLE
FR: Page/Query with all key bindings

### ISSUE THREAD

**Issue by paletochen** [silverbulletmd/silverbullet#641]
Would it be possible to have a page or a query where I could see all the key bindings that Silverbullet uses?

I believe this could help when creating new ones, to see which ones are available.

I am wondering though how much visibility would SB have into the bindings that the browser and/or OS provide.

**Comment by zefhemel** [comment]
Yeah, it won’t be possible to pull the browser’s key bindings. What would be nice is to have `command` as a query source with attributes like:

* `name` command name
* `key` key binding
* `mac` key binding

And perhaps other fields.
